SwiftUIStarter/SwiftUIStarter/NativeDisplayUIViewModel.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  NativeDisplayUIViewModel.swift
+//  SwiftUIStarter
+//
+//  Copyright © 2025 CleverTap. All rights reserved.
+//
+
+import SwiftUI
+import CleverTapSDK
+
+class NativeDisplayUIViewModel: NSObject, ObservableObject, CleverTapDisplayUnitDelegate {
```

Comment:
Rename to `NativeDisplayViewModel` to match the view name.
It might be better to prefix with `CT` as well.